app/serializers/subject_and_class_serializer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# SubjectAndClassSerializer
+class SubjectAndClassSerializer
+  include FastJsonapi::ObjectSerializer
+
+  attributes :id, :created_at
```

Comment:
@josephelias-rubyians  Please include the subject_id and class_id foreign keys. All the columns should be available in the serializer.

Reply:
Yes I have added attributes subject info and class info in the serializer it self.
It will be available in the serializer. Didn't get this.
![image](https://user-images.githubusercontent.com/79498012/114513150-81480280-9c57-11eb-8c42-fb5c0e4b54dc.png)